### ISSUE TITLE
Still any maintenance around here?

### ISSUE THREAD

**Issue by zWhdmB5T** [OlafHaag/chocolatey-packages#29]
Is this package still maintained? Updates available at source site…

**Comment by OlafHaag** [comment]
Hey! I don't actively use chocolate much anymore, nevermind the packages I maintain. So it can take quite some time until I get around to fixing reported issues. I'm happy for anyone taking over maintenance of any of the packages, or opening pull requests!

Is there any specific package that you are referring to?

**Comment by zWhdmB5T** [comment]
@OlafHaag  yes, "JabRef" `JabRef` <https://github.com/OlafHaag/chocolatey-packages/tree/master/automatic/jabref>

[5.11 – 2023-10-22](https://github.com/JabRef/jabref/blob/main/CHANGELOG.md)

**Comment by zWhdmB5T** [comment]
@OlafHaag Thank you for your quick reaction! :-)
